lib/Sema/TypeCheckDecl.cpp

```diff
@@ -874,8 +874,12 @@ static void checkRedeclaration(TypeChecker &tc, ValueDecl *current) {
                         current->getFullName(),
                         otherInit->isMemberwiseInitializer());
         } else {
-          tc.diagnose(current, diag::invalid_redecl, current->getFullName());
-          tc.diagnose(other, diag::invalid_redecl_prev, other->getFullName());
+          tc.diagnoseWithNotes(tc.diagnose(current, diag::invalid_redecl,
+                                           current->getFullName()),
+                               [&]() {
+                                 tc.diagnose(other, diag::invalid_redecl_prev,
+                                             other->getFullName());
+                               });
```

Comment:
We've also been known to single-indent closure bodies from the current indentation level rather than from the square-brackets, as if it were a control-flow construct.

Reply:
👍 , makes sense. I didn't change the other example in CSApply because it made it harder to read.